src/components/EventDashboard/CCPTabPanel.tsx

```diff
@@ -220,13 +287,28 @@ const CCPTabPanel = ({ eventId }: { eventId: string }) => {
         <Add className={classes.buttonIcon} />
         Add CCP
       </Button>
+      {selectedCCP && (
+        <ConfirmModal
+          open={openConfirmDelete}
+          title="Are you sure you want to delete the following CCP?"
+          body={
+            <>
+              <span style={{ color: Colours.Secondary, fontWeight: 500 }}>
+                {(selectedCCP as CCP).name}
+              </span>
+              {' cannot be recovered when deleted.'}
+            </>
+          }
```

Comment:
looking at the other uses of `ConfirmModal`, should we be switching to this pattern for consistency in both code and styling?

Reply:
There's no need to switch to this pattern. This was necessary just because part of the "body" text of the modal/dialog had to be styled differently according to designs.